docs/resources/vpc_public_gateway_dhcp_reservation.md

```diff
@@ -5,15 +5,15 @@ page_title: "Scaleway: scaleway_vpc_public_gateway_dhcp_reservation"
 
 # Resource: scaleway_vpc_public_gateway_dhcp_reservation
 
-Creates and manages the [Scaleway DHCP Reservations](https://www.scaleway.com/en/docs/network/vpc/concepts/#dhcp).
+Creates and manages [Scaleway DHCP Reservations](https://www.scaleway.com/en/docs/network/vpc/concepts/#dhcp).
 
-The static associations are used to assign IP addresses based on the MAC addresses of the Instance.
+These static associations are used to assign IP addresses based on the MAC addresses of the resource.
 
 Statically assigned IP addresses should fall within the configured subnet, but be outside of the dynamic range.
 
-For more information, see [the documentation](https://www.scaleway.com/en/developers/api/public-gateway/#dhcp-c05544) and [configuration guide](https://www.scaleway.com/en/docs/network/vpc/how-to/configure-a-public-gateway/#how-to-review-and-configure-dhcp).
+For more information, see [the API documentation](https://www.scaleway.com/en/developers/api/public-gateway/#dhcp-c05544).
 
-[DHCP reservations](https://www.scaleway.com/en/developers/api/public-gateway/#dhcp-entries-e40fb6) hold both dynamic DHCP leases (IP addresses dynamically assigned by the gateway to instances) and static user-created DHCP reservations.
+[DHCP reservations](https://www.scaleway.com/en/developers/api/public-gateway/#dhcp-entries-e40fb6) hold both dynamic DHCP leases (IP addresses dynamically assigned by the gateway to resources) and static user-created DHCP reservations.
```

Comment:
@yfodil is the part about dynamic DHCP leases still correct, even though this functionality is now on Private Networks?